reports/tuning/Cuemon.DateSpanBenchmark-report-github.md

```
@@ -0,0 +1,41 @@
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)